pkg/crc/api/handlers.go

```diff
@@ -30,15 +30,28 @@ func stopHandler(client machine.Client, crcConfig crcConfig.Storage, _ json.RawM
 	return encodeStructToJSON(commandResult)
 }
 
-func startHandler(client machine.Client, crcConfig crcConfig.Storage, _ json.RawMessage) string {
+func startHandler(client machine.Client, crcConfig crcConfig.Storage, args json.RawMessage) string {
 	startConfig := machine.StartConfig{
-		Name:          constants.DefaultName,
-		BundlePath:    crcConfig.Get(config.Bundle).AsString(),
-		Memory:        crcConfig.Get(config.Memory).AsInt(),
-		CPUs:          crcConfig.Get(config.CPUs).AsInt(),
-		NameServer:    crcConfig.Get(config.NameServer).AsString(),
-		GetPullSecret: getPullSecretFileContent(crcConfig),
-		Debug:         true,
+		Name:       constants.DefaultName,
+		BundlePath: crcConfig.Get(config.Bundle).AsString(),
+		Memory:     crcConfig.Get(config.Memory).AsInt(),
+		CPUs:       crcConfig.Get(config.CPUs).AsInt(),
+		NameServer: crcConfig.Get(config.NameServer).AsString(),
+		Debug:      true,
+	}
+	if args != nil {
```

Comment:
An unit test for this would be good to have :)

Reply:
Done!